{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py

```diff
@@ -3,7 +3,7 @@
 This file is used to configure the behavior of pytest when using the Astropy
 test infrastructure. It needs to live inside the package in order for it to
 get picked up when running the tests inside an interpreter using
-packagename.test
+`{{ cookiecutter.package_name }}.test()`.
```

Comment:
I think it should be module name instead of package name?

Reply:
Fixed.